packages/flutter_tools/test/general.shard/cmake_test.dart

```diff
@@ -38,24 +38,18 @@ void main() {
     final String? name = getCmakeExecutableName(cmakeProject);
 
     expect(name, 'hello');
-  }, overrides: <Type, Generator>{
-    FileSystem: () => fileSystem,
-    ProcessManager: () => processManager,
```

Comment:
Were these overrides just being provided because they were being copied from a previous test? I don't see anywhere in this PR where we are injecting the filesystem and process manager

Reply:
yeah, only a few of these tests actually hit the context

Reply:
As a side note, I'm exploring writing a script that will test what happens if we rename each `testUsingContext` test as `testWithoutContext` and see how many still pass.